xc/device_common.mli

```diff
@@ -44,6 +44,7 @@ val frontend_rw_path_of_device : xs:Xenstore.Xs.xsh -> device -> string
 val frontend_ro_path_of_device : xs:Xenstore.Xs.xsh -> device -> string
 val disconnect_path_of_device : xs:Xenstore.Xs.xsh -> device -> string
 val kthread_pid_path_of_device : xs:Xenstore.Xs.xsh -> device -> string
```

Comment:
Hmm... I should actually get rid of the old function. I'll update the other user of it, on the VIF side as well.

Reply:
Updated that.